src/modules/trash/trash.controller.ts

```diff
@@ -97,27 +97,11 @@ export class TrashController {
   }
 
   @Delete('/all')
-  @HttpCode(204)
+  @HttpCode(202)
   @ApiOperation({
     summary: "Deletes all items from user's trash",
   })
-  async clearTrash(@UserDecorator() user: User) {
-    const { rootFolderId: folderId, id: userId } = user;
-    const deleted = true;
-
-    await Promise.all([
-      await this.fileUseCases
-        .getByFolderAndUser(folderId, userId, deleted)
-        .then((files: Array<File>) =>
-          files.map(this.fileUseCases.deleteFilePermanently),
-        ),
-      await this.folderUseCases
-        .getChildrenFoldersToUser(folderId, userId, deleted)
-        .then((folders: Array<Folder>) =>
-          folders.map(this.folderUseCases.deleteFolderPermanently),
-        ),
-    ]);
-
-    return;
+  clearTrash(@UserDecorator() user: User) {
+    this.trashUseCases.clearTrash(user);
```

Comment:
perhaps you need to use `return` here?

Reply:
I did search it and I couldn't find any reason to have to return void